api4/group.go

```diff
@@ -610,6 +609,11 @@ func getGroups(c *Context, w http.ResponseWriter, r *http.Request) {
 		opts.NotAssociatedToChannel = channelID
 	}
 
+	if getTeamErr != nil && getChannelErr != nil {
```

Comment:
this should be moved up as suggested above.

Reply:
Thanks! Done.